src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+const CHECK_STATUS = './library/categories/CHECK_STATUS';
+
+const stateInit = {
+  type: 'action',
+  checked: true,
+  payload: 'Under construction',
+};
+
+export const checkStatus = (payload) => ({
+  type: CHECK_STATUS,
+  payload,
+});
+
+const catReducer = (state = stateInit, action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return action.payload;
+
```

Comment:
- [ ] In your categories reducer, when the case is `CHECK_STATUS`, it should return a string which says `Under Construction`. Kindly implement it. 👍

Reply:
- [ ] Kindly implement this requirement asked by the previous reviewer